packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -3016,3 +3099,18 @@ function getFromChangeAtomIdMap<T>(
 function setInChangeAtomIdMap<T>(map: ChangeAtomIdBTree<T>, id: ChangeAtomId, value: T): void {
 	map.set([id.revision, id.localId], value);
 }
+
+function areEqualFieldIds(a: FieldId, b: FieldId): boolean {
+	return areEqualChangeAtomIdOpts(a.nodeId, b.nodeId) && a.field === b.field;
+}
+
+function areEqualChangeAtomIdOpts(
```

Comment:
Consider moving that to src\core\rebase\types.ts and using it instead of areEqualCellIds in src\feature-libraries\sequence-field\utils.ts